docs/cvl/expr.md

```diff
@@ -169,6 +169,10 @@ currently undocumented.
    side-effects (such as updating a [ghost variable](ghosts)), only the
    side-effects of the expression that is chosen are performed.
 
+   Regarding the logical operator precedence, `=>` has higher precedence than `<=>`,
+   and unlike math operators both are _right_ associative, so `expr1 => expr2 => expr3`
```

Comment:
I would probably strike "unlike math operators"; I'm not sure that that reference we found was correct, and I'm not sure how universal the conventions are either (see e.g. https://math.stackexchange.com/questions/12223/associativity-of-logical-connectives - several folks there say that the usual convention is right associativity).

Reply:
in all programming languages I know of maath operators are left associative. In particular, math operators in CVL are left associative, so I think the comment is correct.